src/index.js

```diff
@@ -55,6 +55,37 @@ function videoFrameHandler(videoFrame, notifyVideoProcessed, notifyError) {
   // }
 }
 
+async function videoStreamHandler(receivedVideoFrame) {
```

Comment:
Since sample app serves as a public-facing example, we should write a handler that is dedicated for `VideoFrame`, rather than converting it to array buffer and falling back to T1 implementation.

Reply:
updated, created 2 filters that process `VideoFrame` instead of video array buffer